### ISSUE TITLE
SH4 Exception on some ECHELON intros

### ISSUE THREAD

**Issue by benjis8888** [flyinghead/flycast#978]
(retroarch, windows pc)

i've found some games do this when booting but the same games work perfectly on real hardware, by cd and usb gd-rom

one game it's happening on is **Fighting Vipers 2 JP Trainer+16 ECHELON**, but i've had it on other games too, i can provide the image for testing if required

**Comment by flyinghead** [comment]
To be honest I'm not going to spend time on hacked CDI issues. Not until I've fixed all issues on "official" gdi/cue roms.

**Comment by benjis8888** [comment]
Every time I've seen it happen it's always when booting, just after the Dreamcast logo, it's something in the boot code (1st_read.bin) that the emulator doesn't like

**Comment by MastaG** [comment]
@benjis8888 
The intro/cracktro's by echelon/kalistios seem to behave weird on many dc emulators and even on stuff like the DreamShell iso loader.
It's recommended to use the original GDI release instead.

**Comment by benjis8888** [comment]
yes using gdi is ok for the time-being but some people want to experience the old scene releases with the cracktros and trainers, it is part of the nostalgia of the dreamcast. nulldc and redream are running the echelon intros properly without any issues but i prefer to use flycast as it runs through retroarch and has good features such as shaders
